tools/shared/src/main/scala/org/scalajs/core/tools/linker/frontend/BaseLinker.scala

```diff
@@ -137,6 +137,9 @@ final class BaseLinker(semantics: Semantics, esLevel: ESLevel, considerPositions
       assemble(infoInput, getTree, analysis)
     }
 
+    // Make sure we don't export to the same name twice.
+    checkConflictingExports(linkResult, logger, bypassLinkingErrors)
+
```

Comment:
A quick note in case you wonder: It would be nicest to do this in the analysis of course. However, we do not have the names we actually export something under in the Infos, so we can't.
This is maybe something we should consider for 1.0.0. Simply having an analyzer error would be much cleaner.